### PR TITLE
BLD: Fix failing travis builds

### PR DESCRIPTION
**Pull request by ElDeveloper**
Solves the problem of other failing builds.

### CODE REVIEW

**Comment by mortonjt** [comment]
Ready to merge?

File: emperor/qiime_backports/util.py [hunk 1]
```diff
@@ -413,6 +413,7 @@ def _idf(data):
         if n < 3:
             return [numpy_nan,numpy_nan]
         (j,h) = divmod(n/4. + 5/12.,1)
+        j = int(j)
         qlo = (1-h)*x[j-1] + h*x[j]
         k = n - j
         qup = (1-h)*x[k] + h*x[k-1]
```
Comment (mortonjt) [inline emperor/qiime_backports/util.py]: I see that there are a bunch of spacing issues that likely violate pep8.  Is this a problem?

**Comment by mortonjt** [comment]
Minor comments - otherwise 👍 to merge

**Comment by ElDeveloper** [comment]
Thanks!

**Pull request merged**